packages/next/src/server/lib/app-info-log.ts

```diff
@@ -21,31 +21,27 @@ export function logStartInfo({
   maxExperimentalFeatures?: number
 }) {
   Log.bootstrap(
-    bold(
-      purple(
-        ` ${Log.prefixes.ready} Next.js ${process.env.__NEXT_VERSION}${
-          process.env.TURBOPACK ? ' (turbo)' : ''
-        }`
-      )
-    )
+    `${bold(
+      purple(`${Log.prefixes.ready} Next.js ${process.env.__NEXT_VERSION}`)
+    )}${process.env.TURBOPACK ? ' with Turbopack' : ''}`
   )
   if (appUrl) {
-    Log.bootstrap(` - Local:        ${appUrl}`)
+    Log.bootstrap(`- Local:        ${appUrl}`)
   }
   if (networkUrl) {
-    Log.bootstrap(` - Network:      ${networkUrl}`)
+    Log.bootstrap(`- Network:      ${networkUrl}`)
   }
-  if (envInfo?.length) Log.bootstrap(` - Environments: ${envInfo.join(', ')}`)
+  if (envInfo?.length) Log.bootstrap(`- Environments: ${envInfo.join(', ')}`)
 
   if (expFeatureInfo?.length) {
-    Log.bootstrap(` - Experiments (use with caution):`)
+    Log.bootstrap(`- Experiments (use with caution):`)
     // only show maximum 3 flags
     for (const exp of expFeatureInfo.slice(0, maxExperimentalFeatures)) {
-      Log.bootstrap(`   · ${exp}`)
+      Log.bootstrap(`  · ${exp}`)
     }
     /* ${expFeatureInfo.length - 3} more */
     if (expFeatureInfo.length > 3 && maxExperimentalFeatures) {
```

Comment:
Unrelated to the changes in this PR, but shouldn't this be

```suggestion
    if (maxExperimentalFeatures && expFeatureInfo.length > maxExperimentalFeatures) {
```